src/Components/Game/Game.js

```diff
@@ -0,0 +1,353 @@
+import React from 'react';
+
+import CountDownTimer from '../CountdownTimer/CountdownTimer';
+import Score from '../Score/Score';
+import data from '../../data/dictionary.json';
+import './Game.css';
+
+export default class Game extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.INITIAL_STATE = {
```

Comment:
Uppercase constants are only supposed to be used for values that are known prior to program execution. Use uniform naming convention across your entire application.
Refer https://javascript.info/variables#uppercase-constants